test/run-drun/max-stack-variant.mo

```diff
@@ -1,8 +1,8 @@
-//MOC-FLAG --compacting-gc --rts-stack-pages 32 -measure-rts-stack
+//MOC-FLAG -measure-rts-stack
 import { errorMessage; debugPrint; } = "mo:⛔";
 
 actor {
-    let expectedMinimumSize = 31_000;
+    let expectedMinimumSize = 26_000;
```

Comment:
I wonder why the capacity has gone down? Reduction is Rust stack or maybe some additional overhead from the way the global type table is loaded onto the stack for candid-subtype checks? Maybe this will go improve again when that is fixed.

Reply:
--rts-stack-page 32 should provide 2MB of Rust stack, so less than your current allowance of 4MB.... odd.

Reply:
I measured again, and it supports `31_000` (seems to be some old config). However, I believe it should be higher due to the larger stack size. Maybe the optimization with the candid-subtype check would improve this. I'll wait until that optimization, and if it is still now, need to analyze and compare the stack usage...

Reply:
I reverted to `31_000` but it should become higher with the optimization.